helmfile/values/prometheus-operator-sc.yaml.gotmpl

```diff
@@ -290,7 +290,7 @@ prometheus:
       scheme: http
       static_configs:
       - targets:
-        - '{{ requiredEnv "NFS_SC_SERVER_IP" }}:9100'
+        - '{{ .Values.global.nfs_server_ip }}:9100'
```

Comment:
This change (and for wc) should have removed the requirement of exporting the nfs ip environment variable :slightly_smiling_face: